### ISSUE TITLE
Elevated circulating deoxypyridinoline concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6028]
**Preferred term label:** Elevated circulating deoxypyridinoline concentration

**Synonyms** Elevated circulating DPD concentration

**Definition (free text, please give PubMed ID)** Concentration of deoxypyridinoline in the blood is above the normal range., PMID: 9284202

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
